stable/scanengine/templates/deployment.yaml

```diff
@@ -74,6 +74,15 @@ spec:
             value: "{{ .Values.conf.queueArn }}"
           - name: SNS_TOPIC_ARN
             value: "{{ .Values.conf.snsArn }}"
+          - name: "ASYNC_CHECK_CREATION"
+            value: "{{ .Values.conf.enable_async_creation }}"
+          - name: "AWS_SNS_CHECKS_ARN"
+            value: "{{ .Values.conf.sns_checks.topic_arn }}"
+          - name: "AWS_CHECK_CREATOR_WORKERS"
+            value: "{{ .Values.conf.check_creator.num_of_workers }}"
+          - name: "AWS_CHECK_CREATOR_PERIOD"
+            value: "{{ .Values.conf.check_creator.period }}"
+
```

Comment:
- We use camelCase syntax all around
- In order to pass the travis checks the parameters have to exists in the persistence/values.yaml

Reply:
In vulcan/values.yaml:

- In order to keep updated the infra version of the charts add the new sqs / sns to goaws.config.goaws.yaml
- Add the new parameters for readability with the values of the sqs / sns

Reply:
Fixed